libnavui-maneuver/api/current.txt

```diff
@@ -14,13 +7,17 @@ package com.mapbox.navigation.ui.maneuver.api {
   }
 
   public final class MapboxManeuverApi {
-    ctor public MapboxManeuverApi(com.mapbox.navigation.base.formatter.DistanceFormatter formatter, com.mapbox.navigation.ui.maneuver.model.ManeuverOptions maneuverOptions = ManeuverOptions.<init>().build());
```

Comment:
No we are not.
This is the class constructor
```
class MapboxManeuverApi internal constructor(
    private val distanceFormatter: DistanceFormatter,
    private val maneuverOptions: ManeuverOptions,
    private val processor: ManeuverProcessor,
    private val routeShieldApi: MapboxRouteShieldApi
)
```
which is internal as you can see.

This is the constructor overload
```
@JvmOverloads
    constructor(
        formatter: DistanceFormatter,
        routeShieldApi: MapboxRouteShieldApi = MapboxRouteShieldApi(),
        maneuverOptions: ManeuverOptions = ManeuverOptions.Builder().build()
    ) : this(
        formatter,
        maneuverOptions,
        ManeuverProcessor,
        routeShieldApi
    )
```
We are always adding default value for `MapboxRouteShieldApi` as that is the new property exposed here.

Reply:
Actually you did break SemVer. Because with this change Java users who provided both `formatter` and `maneuverOptions` will have to update their projects to include `routeShieldApi` parameter. You should have made this parameter last to avoid breaking SemVer.

Reply:
Resolved.